src/som/interpreter/bc/interpreter.py

```diff
@@ -628,30 +630,30 @@ def interpret(method, frame, max_stack_size):
             )
 
         elif bytecode == Bytecodes.q_super_send_1:
-            invokable = method.get_inline_cache_invokable(current_bc_idx)
-            stack[stack_ptr] = invokable.invoke_1(stack[stack_ptr])
+            invokable = method.get_inline_cache(current_bc_idx)
```

Comment:
Could you rename the `invokable` here and in the following cases to reflect the change? Maybe it could be better called `dispatch_node`.